app/views/post/index.html.erb

```diff
@@ -3,13 +3,20 @@
 <% if @posts.blank? %>
 <p>There are not any posts currently in the system.</p>
 <% else %>
-<p>These are the current posts in our system</p>
+<p class= 'add user'>These are the current posts in our system</p>
 
 <ul id = "posts">
    <% @posts.each do |c| %>
-   <li><%= link_to c.title, {:action => 'show', :id => c.id} %></li>
+   <div class="user-post-preview">
+   <h3>Title: <%= c.title %></h3>
+   <p><%= c.text %></p>
+   <p><%= c.author_id %></p>
+   <p class="post-data">
+     <span>Comments: <%= c.comments_counter %></span>
+     <span>Likes: <%= c.likes_counter %></span>
+   </p>
+ </div>
```

Comment:
- [ ] As mentioned by the previous code reviewer, we are required to display the most recent comments as seen in the screenshot below as compared to ours. We are required to match the wireframe

wireframe
![Screenshot from 2022-12-02 12-45-42](https://user-images.githubusercontent.com/46388113/205296730-6bcd5318-8744-4f53-9710-e08beec59a29.png)

ours
![Screenshot from 2022-12-02 12-46-50](https://user-images.githubusercontent.com/46388113/205296785-40321b13-643d-412a-9885-0abd5990ef4b.png)